hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/utils/RationalizedSigMeta.java

```diff
@@ -146,33 +153,60 @@ public void revokeCryptoSigsFrom(final JKey key) {
                                 : wrappedFn.apply(publicKey);
     }
 
-    public void replacePayerHollowKeyIfNeeded() {
-        try {
-            if (!payerReqSig.hasHollowKey()) return;
-        } catch (NullPointerException npe) {
-            log.warn("payerReqSig not expected to be null", npe);
+    // add javadocs
+    public void replaceHollowKeysIfNeeded() {
+        if (this == NONE_AVAIL || rationalizedSigs == null) {
             return;
         }
 
-        final var targetEvmAddress = payerReqSig.getHollowKey().getEvmAddress();
+        final var hollowKeysInOtherReqKeysIndex = createHollowKeyIndex();
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏This can be moved below just before it's needed at line 185 to reduce it's scope and for an added bonus of not calling `createHollowKeyIndex()` if we don't have `ECDSA_SECP256K1` signatures in `rationalizedSigs`.

Reply:
Could you indicate anew where you suggest moving this initialization --- spotless reformated the files, and I think the numbering is totally different now